include/libcyphal/executor.hpp

```diff
@@ -127,9 +127,8 @@ class IExecutor
             /// according to the last setup.
             ///
             /// @param schedule Contains specifics of how exactly callback will be scheduled.
-            /// @return `true` if scheduling was successful, `false` otherwise (b/c callback has been reset).
             ///
-            virtual bool schedule(const Schedule::Variant& schedule) = 0;
+            virtual void schedule(const Schedule::Variant& schedule) = 0;
```

Comment:
Can't fail at all. Note that, in contrast, `Callback::Any::schedule` still has `bool` return value - the only reason is b/c of possible unbounded_variant without a value (`.has_value() == false`).

Reply:
<sub><sup>*failure is not an option!*</sub></sup>